WooCommerce/Classes/Notifications/ApplicationAdapter.swift

```diff
@@ -40,8 +40,8 @@ extension UIApplication: ApplicationAdapter {
 
     /// Presents a given Message with an "In App" notification
     ///
-    func presentInAppNotification(message: String) {
-        let notice = Notice(title: message, message: nil, feedbackType: .success)
+    func presentInAppNotification(title: String, subtitle: String?, message: String?) {
```

Comment:
We never handled tapping on the Notices, did we? I can receive a notice but when I tap on it, nothing happens. 🤷  Just wanted to confirm.

Reply:
That's correct 😅 I created an issue about this, and I had some other thoughts while writing the issue https://github.com/woocommerce/woocommerce-ios/issues/5330 - would like to hear what you and the team think 🙂